pkg/tfgen/docs.go

```diff
@@ -203,8 +203,8 @@ func getDocsForProvider(g *generator, org string, provider string, resourcePrefi
 // checkIfNewDocsExist checks if the new docs root exists
 func checkIfNewDocsExist(repo string) bool {
 	// Check if the new docs path exists
-	newDocsRoot := filepath.Join(repo, "docs")
-	_, err := os.Stat(newDocsRoot)
+	newDocsPath := filepath.Join(repo, "docs", "resources")
```

Comment:
I am assuming all TF providers have resources. If this isn't a safe assumption we can check if either `docs/resources` or `docs/data-sources` exist

Reply:
all have resources! :)